youtube_dl/extractor/orf.py

```diff
@@ -205,6 +204,8 @@ class ORFFM4IE(ORFRadioIE):
     IE_NAME = 'orf:fm4'
     IE_DESC = 'radio FM4'
     _VALID_URL = r'https?://(?P<station>fm4)\.orf\.at/player/(?P<date>[0-9]+)/(?P<show>4\w+)'
+    api_station = "fm4"
+    loop_station = "fm4"
```

Comment:
Const must be uppercase. Protected must be prefix with underscore. Single quotes.

Reply:
Thanks very much, fixed.